charts/redis/templates/redis-pv-slave.yaml

```diff
@@ -14,8 +14,8 @@
 
 {{- $namespace := include "palisade.namespace" . }}
 {{- $redisname := include "redis.fullname" . }}
-{{- if .Values.global.persistence.statefulSet.pv.enabled }}
-{{- $config := $.Values.global.persistence.redisSlave }}
+{{- if .Values.global.persistence.statefulset.pv.enabled }}
+{{- $config := (index $.Values.global.persistence "redis-slave") }}
```

Comment:
Is there another ticket to change this to redis-worker?

Reply:
yup